vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -1525,6 +1604,17 @@ def get_kv_connector_stats(self) -> Optional[KVConnectorStats]:
             return self.xfer_stats.clone_and_reset()
         return None
 
+    def get_block_ids_with_load_errors(self) -> set[int]:
+        """
+        Return and clear the set of block IDs that failed to load.
+
+        This is called by the scheduler to identify blocks that need
+        to be retried after a NIXL transfer failure.
+        """
+        invalid_blocks = self._invalid_block_ids.copy()
+        self._invalid_block_ids.clear()
```

Comment:
nit: can avoid the copy
```suggestion
        invalid_blocks = self._invalid_block_ids
        self._invalid_block_ids = set()
```